config/routes.rb

```diff
@@ -1,5 +1,8 @@
 Rails.application.routes.draw do
-  # Define your application routes per the DSL in https://guides.rubyonrails.org/routing.html
+  # Adding nested resources routes for users and posts
+  resources :users do
+    resources :posts
+  end 
```

Comment:
- [ ] As stated in the project requirement kindly make sure that you added code to handle only the **4 required URLs.** when I run `rails routes` command in my console, I got the the following output
![route](https://github.com/NitBravoA92/blog-app/assets/93598189/8f5c520b-4f16-431c-913f-4c9240692c46)

- But it should display only the **4 required URLs.** just like the below result 
![route correct](https://github.com/NitBravoA92/blog-app/assets/93598189/85f494ad-ea7e-479c-a0e2-bddccb01272b)

- So kindly modify the` routes.rb` just like the below sample to get the required result 

```
Rails.application.routes.draw do
  # Adding nested resources routes for users and posts
  resources :users, only: [:index, :show] do
    resources :posts, only: [:index, :show]
  end
 

  # Defines the root path route ("/")
  # root "articles#index"
end
```

Reply:
Hello @Abenezer-Tilahun Thank you very much for your suggestions! I have applied the changes.